src/utilities/bcl/templates/ReportingMeasure/measure.rb

```diff
@@ -131,22 +134,43 @@ def run(runner, user_arguments)
       end
     end
 
-    # only try to get the annual timeseries if an annual simulation was run
-    if ann_env_pd
+    # if requested look through time series results
+    if report_drybulb_temp
+
+      # only try to get the annual timeseries if an annual simulation was run
+      if ann_env_pd
+
+        # get desired variable
+        key_value = 'Environment'
+        time_step = 'Hourly' # "Zone Timestep", "Hourly", "HVAC System Timestep"
+        variable_name = 'Site Outdoor Air Drybulb Temperature'
+        output_timeseries = sql_file.timeSeries(ann_env_pd, time_step, variable_name, key_value) # key value would go at the end if we used it.
+
+        if output_timeseries.empty?
+          runner.registerWarning('Timeseries not found.')
+        else
+          runner.registerInfo('Found timeseries.')
+
+          output_timeseries = output_timeseries.get.values
+          min = nil
+          max = nil
+          # loop through timeseries and move the data from an OpenStudio timeseries to a normal Ruby array (vector)
```

Comment:
Don't think that comment is accurate, seems like you're just getting min and max.